Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -328,25 +326,25 @@ bool CBoot::SetupWiiMemory(u64 ios_title_id)
 // Wii Bootstrap 2 HLE:
 // copy the apploader to 0x81200000
 // execute the apploader
-bool CBoot::EmulatedBS2_Wii()
+bool CBoot::EmulatedBS2_Wii(const DiscIO::IVolume* volume)
```

Comment:
That was one of the things that bothered me with the other BS2-related cleanup PRs from last week (mostly next to your "poor mans bootup"), but apparently I forgot about commenting on it.
+1 for `SetupGCMemory`.